tutor/resources/styles/components/media-preview.scss

```diff
@@ -15,7 +18,20 @@
 
   @include tutor-figure();
   @include tutor-tables($tutor-neutral-light, $tutor-neutral-light);
-
+  .os-figure:not(.splash):not(.full-width) {
+    width: 100%;
```

Comment:
Not full-width == 100% width?

Reply:
full-width is actually 100% + 280px so it expands to fill the entire panel